packages/browser/src/engineDetector.ts

```diff
@@ -0,0 +1,12 @@
+import { JavaScriptEngine } from '@backtrace/sdk-core/src/model/data/JavaScriptEngine';
+
+export function getEngine(): JavaScriptEngine {
+    if (navigator.userAgent.includes('Firefox')) {
+        return 'firefox';
+    }
+    if (navigator.userAgent.includes('Safari')) {
```

Comment:
We may want to find a more concrete way of doing this. When I run `navigator.userAgent` in my Chrome console it gives this output: 
`> navigator.userAgent
'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/114.0.0.0 Safari/537.36'` which would return true here.

Reply:
Ok, once again the tactic has changed - this time based on some materials on the Internet I'm verifying If the web browser is Chromium base and if the user agent contains information about Safari. https://www.geeksforgeeks.org/how-to-detect-the-user-browser-safari-chrome-ie-firefox-and-opera-using-javascript/